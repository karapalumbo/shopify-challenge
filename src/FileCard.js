import React from "react";
import { Card } from "@workday/canvas-kit-react/card";
import "./Files.css";

export const FileCard = ({ filename, handleDelete }) => {
  return (
    <Card className="filecard-container">
      <Card.Body>
        <img className="filename" src={`http://localhost:8000/${filename}`} />
      </Card.Body>
      <button onClick={() => handleDelete(filename)}>Remove</button>
    </Card>
  );
};
