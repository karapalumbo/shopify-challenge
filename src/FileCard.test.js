import React from "react";
import { render } from "@testing-library/react";
import { FileCard } from "./FileCard";

it("renders without crashing", function () {
  render(<FileCard />);
});

it("matches snapshot", function () {
  const { asFragment } = render(<FileCard key="123" filename="123.jpg" />);
  expect(asFragment()).toMatchSnapshot();
});
