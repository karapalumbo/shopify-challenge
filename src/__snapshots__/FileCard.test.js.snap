// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="filecard-container css-zhde3j"
  >
    <div
      class="css-6z36tk"
    >
      <img
        class="filename"
        src="http://localhost:8000/123.jpg"
      />
    </div>
    <button>
      Remove
    </button>
  </div>
</DocumentFragment>
`;
